apps/real-time-collaboration/README.md

```diff
@@ -0,0 +1,73 @@
+# Real-time collaboration using Amazon Chime SDK messaging
+
+The Amazon Chime SDK Messaging Yjs demo is a rich text editor that 
+uses Yjs and the messaging features of the Amazon Chime SDK for collaborative editing.
+[Yjs](https://github.com/yjs/yjs) is a network-agnostic CRDT framework for building collaboration features in web applications. 
+In this demo, we use the Amazon Chime SDK messaging to store and exchange shared Yjs data. 
+For more information, see the [Amazon Chime SDK Messaging Yjs blog post]().
+
+## Prerequisites
+
+Before getting started, you must have the following prerequisites:
+
+* An [AWS account](https://aws.amazon.com/free)
+that provides access to AWS services.
+* Ensure that you have installed [Node.js](https://nodejs.org/en/) version 14 or higher.
+
+## Getting started
+
+
+### Creating AWS resources
+
+1. Sign in to the AWS Management Console with your primary account. Switch to the **us-east-1 (N. Virginia)** Region. 
+   Note: The AWS CloudFormation (https://aws.amazon.com/cloudformation/) template in this section needs to be 
+   launched in US east (N. Virginia) Region.
+2. Copy the contents of the AWS CloudFormation template from real-time collaboration sample app in
+   [GitHub](https://github.com/aws-samples/amazon-chime-sdk/blob/main/apps/real-time-collaboration/backend/template.yaml) and 
+   save it in a new file named `real-time-collaboration-app-template.yaml`.
+3. Open the AWS CloudFormation console (https://console.aws.amazon.com/cloudformation/) and choose **Create stack**.
+4. Choose **Upload a template file**, and then browse for `template.yaml` in the `./backend` folder.
+5. For **Stack name**, type `real-time-collaboration-app-demo` and choose **Next**.
+6. On the **Specify Details** page, enter the stack name: **DemoName-ChimeCollabDemo**
+7. Choose **Next**, and then **Next** on the **Configure stack options** page.
+8. On the **Review** page, check the **I acknowledge that AWS CloudFormation might create IAM resources** check box. Then click **Create**.
+9. Creating the stack generates 3 outputs: **ApiGatewayUrl, AppInstanceArn, AdminUserArn**. Note these values 
+   for the outputs to use to configure the app in the next step.
+
+### Install dependencies and build the packages
```

Comment:
Optional: Since other sections start with `-ing`, you can make it to e.g.,

```
Building the packages
```

Reply:
removed it. Accidentally merged this back when I squash the changes.